src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -138,26 +93,64 @@ export class BiggiveMainMenu {
         subSubMenuElement.classList.toggle('display-sub-menu');
       };
     });
+  }
 
-    if (!this.usePresetMenuContent) {
-      this.appendPrimaryNavigationLinks();
-      this.appendNavSecondaryLinks();
-    }
+  private getSecondaryNavLinks() {
```

Comment:
has to be a function to avoid shared nodes - see https://stenciljs.com/docs/templating-jsx#avoid-shared-jsx-nodes . We only previously got away with having a shared node because it wasn't covered by tests.